src/View/index.js

```diff
@@ -0,0 +1,162 @@
+   }
+
+   if (paddingHorizontal && paddingHorizontal[breakpoint]) {
+      styleObject.paddingLeft = paddingHorizontal[breakpoint]
+      styleObject.paddingRight = paddingHorizontal[breakpoint]
+   }
+
+   if (paddingVertical && paddingVertical[breakpoint]) {
+      styleObject.paddingTop = paddingVertical[breakpoint]
+      styleObject.paddingBottom = paddingVertical[breakpoint]
+   }
+
+   return styleObject
+})
+
+export default class View extends React.Component {
+   static propTypes = {
+      alignItems: OIOResponsiveObjectPropType,
+      backgroundColor: OIOResponsiveObjectPropType,
+      backgroundImage: OIOResponsiveObjectPropType,
+      backgroundPosition: OIOResponsiveObjectPropType,
+      backgroundSize: OIOResponsiveObjectPropType,
+      border: OIOResponsiveObjectPropType,
+      borderTop: OIOResponsiveObjectPropType,
+      borderRadius: OIOResponsiveObjectPropType,
+      borderRight: OIOResponsiveObjectPropType,
+      borderBottom: OIOResponsiveObjectPropType,
+      borderLeft: OIOResponsiveObjectPropType,
+      boxShadow: OIOResponsiveObjectPropType,
+      children: PropTypes.node,
+      className: PropTypes.string,
+      display: OIOResponsiveObjectPropType,
+      flex: OIOResponsiveObjectPropType,
+      flexFlow: OIOResponsiveObjectPropType,
+      float: OIOResponsiveObjectPropType,
+      height: OIOResponsiveObjectPropType,
+      justifyContent: OIOResponsiveObjectPropType,
+      margin: OIOResponsiveObjectPropType,
+      marginTop: OIOResponsiveObjectPropType,
+      marginLeft: OIOResponsiveObjectPropType,
+      marginRight: OIOResponsiveObjectPropType,
+      marginBottom: OIOResponsiveObjectPropType,
+      minHeight: OIOResponsiveObjectPropType,
+      maxHeight: OIOResponsiveObjectPropType,
+      minWidth: OIOResponsiveObjectPropType,
+      maxWidth: OIOResponsiveObjectPropType,
+      onClick: PropTypes.func,
+      onKeyUp: PropTypes.func,
+      onScroll: PropTypes.func,
+      order: OIOResponsiveObjectPropType,
+      overflow: OIOResponsiveObjectPropType,
+      padding: OIOResponsiveObjectPropType,
+      paddingTop: OIOResponsiveObjectPropType,
+      paddingLeft: OIOResponsiveObjectPropType,
+      paddingRight: OIOResponsiveObjectPropType,
+      paddingBottom: OIOResponsiveObjectPropType,
+      paddingHorizontal: OIOResponsiveObjectPropType,
+      paddingVertical: OIOResponsiveObjectPropType,
+      position: OIOResponsiveObjectPropType,
+      scroll: OIOResponsiveObjectPropType,
+      style: PropTypes.object,
+      textAlign: OIOResponsiveObjectPropType,
+      top: OIOResponsiveObjectPropType,
+      left: OIOResponsiveObjectPropType,
+      right: OIOResponsiveObjectPropType,
+      bottom: OIOResponsiveObjectPropType,
+      WebkitOverflowScrolling: OIOResponsiveObjectPropType,
+      width: OIOResponsiveObjectPropType,
+      zIndex: OIOResponsiveObjectPropType
+   }
+
+   static defaultProps = {
+      className: '',
+      display: r`block`,
+      onClick: () => {},
```

Comment:
I would avoid empty event handlers if they're not needed, especially with events as frequents as these, on components used as frequently as View

Reply:
I gave an example of what you might do instead in the `render` fn below